docs/src/main/asciidoc/preface.adoc

```diff
@@ -9,8 +9,7 @@ linear fashion or you can skip sections if something does not interest you.
 
 == About the documentation
 The Spring Cloud Task reference guide is available in {spring-cloud-task-docs}[html] 
-and {spring-cloud-task-docs}/index.pdf[pdf],
-{spring-cloud-task-docs}/index.epub[epub] . The
+and {spring-cloud-task-docs}/index.pdf[pdf] . The
```

Comment:
Thank you for your contribution.   However there is a  space between [pdf] and the period.   Can you please remove.

Reply:
Thank you for resolving the issue.